lib/component-utils/snabbdom-params-module.js

```diff
@@ -0,0 +1,13 @@
+function updateParams(oldVnode, vnode) {
+  const elm = vnode.elm;
+  // common guards used in snabbdom-attr-module
+  const oldParams = oldVnode.data.params;
```

Comment:
added this guard just like attr module in snabbdom 

@themacguffinman, @tdumitrescu thinks that following snabbdom-attr-module's pattern to have some null check guards on the top may boost perf / more battle proved.

Reply:
Fair enough, the attr module's guards seem more correct now that I took a second look at it. Sorry for leading you astray on this.